utils/node_local_test.py

```diff
@@ -917,7 +933,7 @@ def stop(self, wf):
         self.conf.compress_file(self.control_log.name)
 
         for log in self.server_logs:
-            log_test(self.conf, log.name, leak_wf=wf)
+            log_test(self.conf, log.name, leak_wf=wf, skip_fi=self._fi)
```

Comment:
Is this backward? In `__init__`: `self._fi = enable_fi`. So `skip_fi=not self._fi` ?

Reply:
I think the skip_fi is probably named backwards.  `skip_fi` means do not warn on warnings in the file related to fault injection, therefore if fault injection is on and therefore expected we should not warn if it happens.  I have more work on this (not least finding and reporting issues) so I can re-name or re-work this at a later date.